Doc/whatsnew/3.12.rst

```diff
@@ -107,6 +107,14 @@ Deprecated
 Removed
 =======
 
+:class:`configparser.ParsingError` no longer has a ``filename`` attribute or
+constructor argument.  Use the ``source`` attribute and argument name instead.
+
+:mod:`configparser` no longer has a ``SafeConfigParser`` class, use the shorter
+:class:`~configparser.ConfigParser` name instead.
+
+:class:`configparser.ConfigParser` no longer has a ``readfp`` method, use
+:meth:`~configparser.ConfigParser.read_file` instead.
```

Comment:
Should these refer to the relevant GH issue?

(Compare https://github.com/python/cpython/pull/28292/files.)

(gh-89336 is in this PR title, although that was originally the 3.11 removal which became a postponement. I had suggested in https://github.com/python/cpython/issues/89336#issuecomment-1094366625 to create a new issue for 3.12 removal, but I personally don't mind if the issue number is recycled.)